corehq/apps/reports/standard/deployments.py

```diff
@@ -92,10 +92,25 @@ def headers(self):
                              prop_name='reporting_metadata.last_submissions.commcare_version',
                              alt_prop_name='reporting_metadata.last_submission_for_user.commcare_version',
                              sql_col='last_form_app_commcare_version'),
-        )
+        ]
+
+    @property
+    def headers(self):
+        columns = self._columns
+        if self.show_build_profile:
+            columns.append(
+                DataTablesColumn(_("Build Profile"),
+                                 help_text=_("The build profile from the user's last hearbeat request."),
+                                 sortable=False)
+            )
+        headers = DataTablesHeader(*columns)
         headers.custom_sort = [[1, 'desc']]
         return headers
 
+    @cached_property
+    def show_build_profile(self):
+        return toggles.SHOW_BUILD_PROFILE_IN_APPLICATION_STATUS.enabled(self.domain)
```

Comment:
Could this use `domain_has_privilege(self.domain, privileges.BUILD_PROFILES)` instead of a new feature flag?

Reply:
that will make it available to all domains based on subscription?

Reply:
@calellowitz probably knows the answer. I'm not 100% sure how build profile privileges work, I'm always surprised when I remember it's not a feature flag. But that's the logic we use to control whether or not to display the build profiles tab on the releases page, so I'd think it would also be appropriate to use here.

Reply:
from the code it look like it depends on the subscription, which looks like the right thing for showing build profiles just like features that are available for a subscription but this one is where we want to control who can access it irrespective of subscription. I am guessing we wont enable it for a domain that don't even have build profiles permissions so the FF should suffice.

Reply:
cool, makes sense